engine/runtime-integration-tests/src/test/scala/org/enso/compiler/test/context/SuggestionBuilderTest.scala

```diff
@@ -245,6 +245,80 @@ class SuggestionBuilderTest extends AnyWordSpecLike with Matchers {
       )
     }
 
+    "build method with an ascribed argument" in {
+
+      val code =
+        """import Standard.Base.Data.Text.Text
+          |
+          |foo (a : Text) = 42""".stripMargin
+      val module = code.preprocessModule
+
+      build(code, module) shouldEqual Tree.Root(
+        Vector(
+          ModuleNode,
+          Tree.Node(
+            Suggestion.DefinedMethod(
+              externalId = None,
+              module     = "Unnamed.Test",
+              name       = "foo",
+              arguments = Seq(
+                Suggestion.Argument(
+                  "a",
+                  "Standard.Base.Data.Text.Text",
+                  false,
+                  false,
+                  None
+                )
+              ),
+              selfType      = "Unnamed.Test",
+              returnType    = SuggestionBuilder.Any,
+              isStatic      = true,
+              documentation = None,
+              annotations   = Seq()
+            ),
+            Vector()
+          )
+        )
+      )
+    }
+
+    "build method with an ascribed and default argument" in {
+
+      val code =
+        """import Standard.Base.Data.Text.Text
+          |
+          |foo (a : Text = "42") = a""".stripMargin
```

Comment:
btw. what happens if there are both kinds of ascription and they don't match?
```
foo : Any -> Any
foo (a : Text) = a
```

(I guess ideally we should get a warning from the compiler, but that's another topic; but here I'm interested what is resolved for the suggestions DB)